src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    enum Colors {White, Yellow, Orange, Red, Magenta, Green, Blue, Pink, Black}
+
+    public static String getRandomColor() {
```

Comment:
```suggestion
    public String getRandomColor() {
```

this method should not be static